src/shared/directives/conditional-show/domain/conditional-show.model.spec.ts

```diff
@@ -538,7 +539,7 @@ describe('conditional-show', () => {
 
     it('should evaluate AND conditions correctly for a mix of EQUALS and CONTAINS', () => {
       caseField2.value = ['s4', 's2', 's3'];
-      let sc = new ShowCondition('field4="s1 AND s2" AND field2CONTAINSs3,s4');
+      let sc = new ShowCondition('field4="s1 AND s2" AND field2 CONTAINS "s3,s4"');
```

Comment:
Same as the comment above about operators not needing to have spaces before or after...

Reply:
As I mentioned in my earlier comment